web/styles/zulip.css

```diff
@@ -837,6 +837,10 @@ strong {
 
             .typeahead-text-container {
                 align-self: center;
+                max-width: 260px;
```

Comment:
Why 260px?

Reply:
Currently, only a maximum of 35 characters are shown in the description which approximately makes the width of the list-item to 260px but that depends on the length of the stream name.

Reply:
Since we are introducing variable font-size, this approximation method won't work, but this equates to `292px` overall width of the typeahead which looks good to me since we want to keep it under `320px`.

Reply:
Let's add a comment here.